test/test_transforms.py

```diff
@@ -457,26 +455,25 @@ class TestPad:
     def test_pad(self):
         height = random.randint(10, 32) * 2
         width = random.randint(10, 32) * 2
-        img = torch.ones(3, height, width)
+        img = torch.ones(3, height, width, dtype=torch.uint8)
         padding = random.randint(1, 20)
         fill = random.randint(1, 50)
         result = transforms.Compose(
             [
                 transforms.ToPILImage(),
                 transforms.Pad(padding, fill=fill),
-                transforms.ToTensor(),
+                transforms.PILToTensor(),
             ]
         )(img)
         assert result.size(1) == height + 2 * padding
         assert result.size(2) == width + 2 * padding
         # check that all elements in the padded region correspond
         # to the pad value
-        fill_v = fill / 255
         eps = 1e-5
         h_padded = result[:, :padding, :]
         w_padded = result[:, :, :padding]
-        torch.testing.assert_close(h_padded, torch.full_like(h_padded, fill_value=fill_v), rtol=0.0, atol=eps)
-        torch.testing.assert_close(w_padded, torch.full_like(w_padded, fill_value=fill_v), rtol=0.0, atol=eps)
+        torch.testing.assert_close(h_padded, torch.full_like(h_padded, fill_value=fill), rtol=0.0, atol=eps)
```

Comment:
Does assert_close + eps make sense  here? Looks like the image is now uint8?

Reply:
Good call changed to 0.